packages/gitbook/src/lib/api.ts

```diff
@@ -208,6 +208,78 @@ export const getUserById = cache({
     },
 });
 
+/**
+ * Get the latest version of an OpenAPI spec by its slug.
+ */
+export const getLatestOpenAPISpecVersion = cache({
+    name: 'api.getLatestOpenApiSpecVersion',
+    tag: (organization, openAPISpec) =>
+        getAPICacheTag({
+            tag: 'openapi',
+            organization,
+            openAPISpec,
```

Comment:
```suggestion
            slug,
```

it's the slug right?

Reply:
Yep and it's the organization ID, since we like not being precise here I continue the on the same vein 😅